wazuh/wazuh-kibana-app
# Add process name in ports information for Linux systems

### Description
Added columns for linux agents in the Network ports table in inventory data, also added in the reports. The order of the columns has been modified so that the ports are the first one.
 
 I have changed the way to search for an agent since I was using the `q` parameter and in this issue we were told that to get an agent we should use `agents?agent_list=` since it was faster.
 
### Issues Resolved

- #5529

### Evidence

Linux

![image](https://github.com/wazuh/wazuh-kibana-app/assets/63758389/c2cbb452-64ca-44bb-9d57-a73950e00304)

![image](https://github.com/wazuh/wazuh-kibana-app/assets/63758389/b5cb4446-aba7-472f-aba3-ff0865eb93ad)

### Test

1. Navigate to agents 
2. Select a linux agent and navigate to inventory data
3. Check that the process and PID columns were added in the network ports table
4. Generate a report and check that columns were added
5. In both cases, the port column must be the first one.
6. Check that there are no column changes in other operating systems

### Check List
